integrationtest/vert.x/pom.xml

```diff
@@ -307,12 +281,10 @@
                             </modules>
                             <module>
                                 <mainClass>com.example.HelloWorldServer</mainClass>
-                                <moduleInfo>
-                                    <name>com.example</name>
-                                    <exports>
-                                        com.example to io.vertx.core;
-                                    </exports>
-                                </moduleInfo>
+                                <moduleInfoSource>open module com.example {
```

Comment:
Does the module have to be open?

Reply:
Same answer here.
The main change is in the `<jdepsExtraArgs>` that allows jdeps to be less dumb by analyzing modules already created by the plugin. It's the reason why I change some module order : to have dependent module computed by jdeps after upstream ones

Reply:
Have you tested the jlink image @aalmiray ? I'm not saying that my config is totally ok but it produces a working result. I'm not expert enough in Vert.x to go further : we should have a more complex app example to fine tune the config automatically computed by `jdeps`

Reply:
Granted, the use of `open` makes everything in the module visible to outsiders. The error you see could also be fixed by explicitly exporting the required package to the vertx module. 

Open modules should not be used by libraries but I suppose it's the easy way for applications ad it's this case.

Reply:
So I should change it to an export mention I guess. Will try that